Integration tests fail on intermittent `deis run` error

This is a manifestation of #1516 in the tests.  Test output:

```
apps:run echo hello

503 SERVICE UNAVAILABLE
could not attach to container

error at command wait
--- FAIL: TestApps (135.71 seconds)
    itutils.go:206: exit status 1
```

Relevant controller logs:

```
2014-09-29 07:49:25 [169] [DEBUG] POST /api/apps/appssample/run
INFO appssample: test runs 'echo hello'
ERROR appssample: appssample_v2.admin.1 (run): could not attach to container
```

--- a/controller/scheduler/coreos.py
+++ b/controller/scheduler/coreos.py
@@ -281,7 +281,7 @@ class FleetHTTPClient(object):
             return rc, output
 
         # wait for container to start
-        for _ in range(30):
+        for _ in range(1200):
             rc, _ = _do_ssh('docker inspect {name}'.format(**locals()))
             if rc == 0:
                 break

